### ISSUE TITLE
Breadcrumbs helper: Breadcrumbs are only visible in current view.

### ISSUE THREAD

**Issue by atalayk** [cakephp/cakephp#10387]
This is a (multiple allowed):

* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* CakePHP Version: 3.4
* Platform and Target: LAMP (PHP 7)

### What you did
I have added breadcrumbs in /Authors/view.ctp
$this->Breadcrumbs->add(
    'Authors',
    ['controller' => 'Authors', 'action' => 'view',2]
);

After a redirect to /Records/Edit/3
Inside /Records/edit.ctp  I created a "BACK" button to enable moving back
echo $this->Breadcrumbs->render(
    ['class' => 'breadcrumbs-trail'],
    ['separator' => '<i class="fa fa-angle-right"></i>']
);

But breadcrumbs are empty.
If I get the breadcrumbs in /Authors/view.ctp (right after I add them then it works)

Basically when focus moves to another view, breadcrumbs are lost.

**Comment by ADmad** [comment]
The crumbs are stored in memory by the helper. They don't persist across requests / pages.
